src/models/continuity/ContGateTurn.ts

```diff
@@ -17,7 +18,12 @@ export default class ContGateTurn
 
   duration = 8;
 
-  centerPoints: [number, number][] = [];
+  centerPoints: Map<number, [number, number]> = new Map<
+    number,
+    [number, number]
+  >();
```

Comment:
Can you leave a comment to explain that the key of the map is each stuntsheet dot's id? Also what is contained in value of the map?